src/main/java/cloud/eppo/EppoHttpClient.java

```diff
@@ -41,34 +41,22 @@ private static OkHttpClient buildOkHttpClient() {
     return builder.build();
   }
 
-  // TODO: use this for Java, callback for Android; clean as needed
-  public Response get(String path) {
-    HttpUrl httpUrl =
-        HttpUrl.parse(baseUrl + path)
-            .newBuilder()
-            .addQueryParameter("apiKey", apiKey)
-            .addQueryParameter("sdkName", sdkName)
-            .addQueryParameter("sdkVersion", sdkVersion)
-            .build();
+  public byte[] get(String path) {
```

Comment:
pass around bytes unless a string is absolutely needed. The JSON parser takes bytes, so we save an unneeded conversion step

Reply:
there's a tradeoff here in which we can't stream the data and have to load it all in memory. seems fine given it doesn't seem like it's a massive amount of data, but something to keep in mind.

Reply:
Alternatively you could return an `InputStream` or equivalent, as long as you ensure the caller closes it properly